app/src/main/java/com/jerboa/ui/components/home/BottomNavActivity.kt

```diff
@@ -180,11 +197,13 @@ fun BottomNavActivity(
             Scaffold(
                 snackbarHost = { JerboaSnackbarHost(snackbarHostState) },
                 bottomBar = {
-                    if (appSettings.showBottomNav) {
+                    if (appSettings.showBottomNav && account.isReady()) {
```

Comment:
Unfortunately, this means an `anon` account will not show the bottom bar.

I couldn't find a way around this, because even tho I'm persisting `is_admin` to the DB, I still need to wait for it to read the database, otherwise the registration button will pop up in the list.

IE, even if I do `if (appSettings.showBottomNav && (account.isAnon() || account.isReady())`, it means that the bottom bar will show 5 tabs, then *suddenly* the 6th tab will pop up only after the DB `isAdmin` status is loaded.

Reply:
I putting this on hold rn, until I have some time to look into this. Because this should definitely be possible in a way.

Because this has bigger consequences. It would make those navigation actions things completely inaccessible. Rn its either bottomnav or in the side panel. I'll probably have tomorrow time for this

Reply:
No probs. As a simple fix I could also remove that `account.isReady()` check. Its mainly just a visual annoyance that these extra bottom bar icons suddenly pop up, but that could be tackled at a later time.

Reply:
Anon accounts are never ready thus isReady check fails for them. (Made it that way because this allowed anon Accounts go to through the checks and display "Not logged in" toasts )